client/src/node/main.ts

```diff
@@ -602,4 +602,19 @@ export class SettingMonitor {
 			void this._client.stop();
 		}
 	}
-}
\ No newline at end of file
+}
+
+function handleChildProcessStartError(process: ChildProcess, message: string) {
+	if (process == null) {
+		return Promise.reject<MessageTransports>(message);
+	}
+
+	return new Promise<MessageTransports>((_, reject) => {
+		process.on("error", (err) => {
+			reject(`${message} ${err}`);
+		});
```

Comment:
See https://nodejs.org/api/child_process.html#event-error

It's unfortunate that this is an event instead of it just being thrown from `spawn`.